src/PairedUrlStructure/QueryVarUrlStructure.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Class QueryVarUrlStructure.
+ *
+ * @package AmpProject\AmpWP
+ */
+
+namespace AmpProject\AmpWP\PairedUrlStructure;
+
+use AmpProject\AmpWP\PairedUrlStructure;
+
+/**
+ * Descriptor for paired URL structures that include the `?amp=1` query parameter.
+ *
+ * @todo Rename to QueryArg for better parity with add_query_arg(). Or QueryParam parameter for better spec alignment?
```

Comment:
I'd go with `QueryParam`.

Reply:
Humm. I just went through renaming it in d29f652de8cc6752d64fbda865ff5d9c13326ce2, but then I'm seeing references to `query_var` all over the place for this idea, including the `QueryVar` interface. Would not `query_var` be more consistent in the codebase instead of `query_param`?

Reply:
Oh I didn't know you meant for it to be such as extensive modification; I thought it was limited to the name of the class (and probably what's within it, and its references).

I think using `query_var` would work just fine. As long as a comment or description is added in places where uncertainty is bound I don't see any issues with it.

Reply:
Reverted in 651b41155a0078fb4afcc1c7ac44b582dd9c430b